libraries/AP_BattMonitor/tests/test_calculate_mah.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016  Intel Corporation. All rights reserved.
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Where did you pull this license info from? The Intel part is very very odd and should be removed?
```suggestion
/*
 */
```

From my understanding the GNU GPL3 license only appears at the very top of the tree and a few other files such as top-level vehicle files.

Reply:
Ignore me just learned a bit about this stuff.

Reply:
Whoops, I blindly copied the AP_GPS test file and didn't even notice that Intel GNU license. I'll clean that it up, thanks!

Reply:
the intel copyright should be removed

Reply:
done